README.md

```diff
@@ -9,6 +9,16 @@ Firstly, this project is built in Python 3.6+ and JavaScript (ES6). You need to
 
 Secondly, it is imperative you use a virtual env (instead of your system Python) to use/contribute to the project, else things could get messy.
 
+### Cloning and Pulling
+
+We are using git submodules in `robotics-prototype`. This means that we are using code that is external to our repository. To ensure that it also downloads all the packages, use the commands below :
+
+Clone : `git clone --recursive https://github.com/space-concordia-robotics/robotics-prototype`
+
+Pull : `git pull; git submodule update --init --recursive`
+
+If you do not have access to the GitLab repository, you will not be able to successfully authenticate to clone `rover2018-elec`. This is fine as long as you do not need the PDS code.
```

Comment:
We could also add two helpful links here:
1. [Setting up ssh key](https://docs.gitlab.com/ee/gitlab-basics/create-your-ssh-keys.html)
2. The link to the stackoverflow question you sent me addressing Creating Access Token and using this to authenticate during pulling of elec repo, instead of regular account password.

Reply:
done